pkg/chartrenderer/default.go

```diff
@@ -89,43 +93,37 @@ func (r *chartRenderer) RenderEmbeddedFS(embeddedFS embed.FS, chartPath, release
 	return r.renderRelease(chart, releaseName, namespace, values)
 }
 
-func (r *chartRenderer) renderRelease(chart *chartapi.Chart, releaseName, namespace string, values interface{}) (*RenderedChart, error) {
-	chartName := chart.GetMetadata().GetName()
-
+func (r *chartRenderer) renderRelease(chart *helmchart.Chart, releaseName, namespace string, values interface{}) (*RenderedChart, error) {
 	parsedValues, err := json.Marshal(values)
 	if err != nil {
-		return nil, fmt.Errorf("can't parse variables for chart %s: ,%s", chartName, err)
+		return nil, fmt.Errorf("can't parse variables for chart %s: ,%s", chart.Metadata.Name, err)
 	}
-	chartConfig := &chartapi.Config{Raw: string(parsedValues)}
-
-	err = chartutil.ProcessRequirementsEnabled(chart, chartConfig)
+	valuesCopy, err := chartutil.ReadValues(parsedValues)
 	if err != nil {
-		return nil, fmt.Errorf("can't process requirements for chart %s: ,%s", chartName, err)
+		return nil, fmt.Errorf("can't read variables for chart %s: ,%s", chart.Metadata.Name, err)
 	}
-	err = chartutil.ProcessRequirementsImportValues(chart)
-	if err != nil {
-		return nil, fmt.Errorf("can't process requirements for import values for chart %s: ,%s", chartName, err)
+
+	if err := chartutil.ProcessDependencies(chart, valuesCopy); err != nil {
+		return nil, fmt.Errorf("can't process chart %s: ,%s", chart.Metadata.Name, err)
```

Comment:
Not introduced with this PR but we could use `%w` instead of `,%s`. I don't know why the latter is used.

```suggestion
		return nil, fmt.Errorf("failed to process chart %s: %w", chart.Metadata.Name, err)
```